packages/gasket-plugin-metadata/EXAMPLES.md

```diff
@@ -0,0 +1,148 @@
+# @gasket/plugin-metadata Examples
+
+This document provides working examples for all exported interfaces and functions from `@gasket/plugin-metadata`.
+
+### Basic Plugin Setup
+
+```js
+// gasket.js
+import { makeGasket } from '@gasket/core';
+import pluginMetadata from '@gasket/plugin-metadata';
+
+export default makeGasket({
+  plugins: [
+    pluginMetadata
+  ]
+});
+```
+
+## metadata
+
+Customize plugin metadata at runtime.
+
+```js
+// Basic metadata customization
+export default {
+  name: 'gasket-plugin-example',
+  hooks: {
+    metadata(gasket, data) {
+      return {
+        ...data,
+        // Add custom metadata
```

Comment:
Move this customField example to the following complex example. It's very atypical to have custom properties